lib/cucumber/tcl/framework.tcl

```diff
@@ -56,11 +65,14 @@ proc _search_steps {step_name {execute 0}} {
   global STEPS
 
   foreach step $STEPS {
-    set existing_step_name [lindex $step 0]
-    set existing_step_body [lindex $step 1]
+    set existing_step_name   [lindex $step 0]
+    set existing_step_params [lindex $step 1]
+    set existing_step_body   [lindex $step 2]
+
 
-    if {[regexp $existing_step_name $step_name matchresult]} {
+    if {[regexp $existing_step_name $step_name matchresult {*}[join $existing_step_params]]} {
       if {$execute == 1} {
+#	puts "JOWERS -> $existing_step_name, $step_name, $matchresult"
```

Comment:
No! I really need a pre-commit hook that blocks checkin if it finds JOWERS anywhere in the code. I've removed that and will push up the change shortly.